src/components/FundCardNew/FundStatus/index.tsx

```diff
@@ -45,7 +45,7 @@ const FundStatus = ({ day, monthly, tvl }: IFundStatusProps) => {
       </S.FundStatus>
       {FundStatusList.map(item => (
         <S.FundStatus key={item.name}>
-          {item?.value ? (
+          {item?.value || item?.value === 0 ? (
```

Comment:
Checking for `!= null` already checks for both `undefined` or `null`

```suggestion
          {item?.value != null ? (
```